help/access/file-shares.md

```diff
@@ -1,302 +1,237 @@
----
-title: File shares. File Manager
----
+# Files
 
-**File Manager** lets you work with files and directories on your system from the convenience of your web browser. You
-can browse, preview, open, create, delete, rename, download, clone, and share files and directories.
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
 
-To open **File Manager**, from the main menu, select **Data**>**Files**.
+Datagrok offers a convenient interface for working with files. You can connect to [popular file systems](supported-connectors.md), including the [Amazon S3 bucket](connectors/s3.md), [Dropbox](connectors/dropbox.md), [Google Drive](connectors/googlecloud.md), and [Git](connectors/git.md), as well as [Windows and Linux network shares](connectors/files.md). In addition, each Datagrok user has a personal directory called **Home** for uploading files using the drag-and-drop feature. Datagrok automatically creates this directory upon signup.
 
-In this article:
+:::note
 
-* [Getting data](#getting-data)
-* [Sharing files](#sharing-files)
-* [Manage file shares](#manage-file-shares)
-* [Manage files](#manage-file-shares)
-* [Preview file and folders](#browse-and-preview)
-* [Customizations](#customizations)
-* [Automation](#automation)
+Connecting to an SMB file storage is only available for on-premise deployment and is not available on the public Datagrok instance (public.datagrok.ai).
 
-> Key concept: _file share_
->
-> We use the term _file share_ when referring to the following:
->
->* Connections to folders and files located on a Datagrok server that you access with a Datagrok client.
->* Any folder or file in the **File Manager** that has been shared with others.
->
->_File shares_ are entities<!--inseert a link-->, meaning you can perform a standard set of operations against them (for
-> example,
-> annotate, set access privileges, [use in automation workflows](data-pipeline.md), or enable discovery by other
-> Datagrok
-> users).
->
-> Note: For enterprise users, the Datagrok administrator defines which file shares you can access and which privileges
-> you have in them.
+:::
 
-## Getting data
+:::note developers
 
-Each Datagrok user has a personal directory for files and folders they want to upload directly to Datagrok. This
-personal directory is accessible in the **Folder Tree** under the name _Home_. The **Home** directory is created
-automatically upon signup. By default, files and folders in your **Home** directory are visible to you only.
+You can [create custom connectors](create-custom-connectors.md) and [read files programmatically](../develop/how-to/access-data.md/#reading-files).
 
-In addition, you can connect network folders hosted on supported data sources, and access files and directories shared
-with you.
+:::
 
-### Supported data sources
+## Connecting to file storage
```

Comment:
Too many topics to my taste. "Connecting to file storage", "Adding connection", "Modifying connection"... I suggest to just have one  "Connecting to file storage" and then just mention that you can edit it by right-clicking and choosing "Edit"